src/GraphQL/Resolver/RichTextResolver.php

```diff
@@ -34,4 +34,11 @@ public function xmlToHtml5Edit(DOMDocument $document)
     {
         return $this->richTextEditConverter->convert($document)->saveHTML();
     }
+
+    public function xmlToPlainText(DOMDocument $document)
+    {
+        $html = $this->richTextConverter->convert($document)->saveHTML();
+
+        return strip_tags($html);
```

Comment:
I'm not positive but using `$domDocument->textContent` might be better for this.

Reply:
Oh. That is a good catch, I'll give it a try :D

Reply:
The result looks good, but it doesn't strip HTML comments you get in debug mode.

```
the most tasty.\n  \n<!-- START vendor/ezsystems/ezpublish-kernel/eZ/Bundle/EzPublishCoreBundle/Resources/views/FieldType/RichText/embed/content.html.twig (EzPublishCoreBundle:FieldType/RichText/embed:content.html.twig) -->\n    <div class=\"align-right ez-embed-type-image\">\n    \n
```

Reply:
Alright, :+1: from me then.